frameworks/helloworld/tests/test_taskcfg.py

```diff
@@ -31,21 +33,27 @@ def test_deploy():
 
     # we can get brief blips of TASK_RUNNING but they shouldnt last more than 2-3s:
     consecutive_task_running = 0
-    def fn():
+
+    @retrying.retry(
+        wait_fixed=1000,
+        stop_max_delay=wait_time*1000,
+        retry_on_exception=lambda ex: False,
+        retry_on_result=lambda res: res is False)
```

Comment:
Does returning `False` mean we should `retry` in this case? Would removing the `return` statement not have the same effect?

Reply:
That is assuming that retrying.retry works like shakedown.wait_for. Be default, all returns are valid; you must specific that return False is a retry.